publish/publish.proj

```diff
@@ -215,32 +215,17 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="FindDownloadedPackagesForSigning">
-    <PropertyGroup>
-      <!-- Glob matching packages that we want to embed symbol signatures in. Used in BuildTools. -->
-      <SymbolPackagesToPublishGlob Condition="'$(SymbolPackagesToPublishGlob)' == ''">$(DownloadDirectory)**\*.symbols.nupkg</SymbolPackagesToPublishGlob>
-    </PropertyGroup>
-
+  <Target Name="PublishSymbols"
+          DependsOnTargets="RestoreOptionalToolingPackages" 
+	  Condition="'$(SymbolServerPath)'!=''" >
     <ItemGroup>
-      <SymbolPackagesToEmbedIndex Include="$(SymbolPackagesToPublishGlob)" />
-      <NormalPackagesToNotEmbedIndex Include="$(DownloadDirectory)**\*.nupkg"
-                                     Exclude="@(SymbolPackagesToEmbedIndex)" />
+      <SymbolPackagesToPublish Include="$(DownloadDirectory)**\*.symbols.nupkg" />
     </ItemGroup>
-
-    <!-- Copy any nupkgs we skip running EmbedIndex on directly to the output folder. -->
-    <Copy SourceFiles="@(NormalPackagesToNotEmbedIndex)"
-          DestinationFolder="$(PublishDirectory)" />
-  </Target>
-
-  <Target Name="SignSymbolPackages"
-          DependsOnTargets="FindDownloadedPackagesForSigning;InjectSignedSymbolCatalogIntoSymbolPackages"
-          Condition="'$(WindowsSdkDir)' != '' AND '$(SignType)' == 'real'">
-    <ItemGroup>
-      <EmbedIndexArgPairs Include="@(SymbolPackagesToEmbedIndex -> '%(Identity) $(PublishDirectory)%(Filename)%(Extension)')" />
-    </ItemGroup>
-
-    <!-- Inline the EmbedIndex.ps1 script used in CoreFX and CoreCLR: directly call EmbedIndex. -->
-    <Exec Command="$(DotNetToolDir)\dotnet $(EmbedIndexToolDir)\tools\EmbedIndex.dll %(EmbedIndexArgPairs.Identity)" />
+    <PropertyGroup>
+      <_SymbolPackagesToPublish>@(SymbolPackagesToPublish)</_SymbolPackagesToPublish>
+    </PropertyGroup>
+    <MSBuild Projects="$(MSBuildThisFileDirectory)\publish-symbols.proj" 
```

Comment:
Instead of doing this I would prefer we split out the symbol publishing step by it self in the publish build definition leg.